prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -314,6 +314,57 @@ presubmits: # runs on PRs
             effect: NoSchedule
         nodeSelector:
             dedicated: "high-cpu"
+    - name: pre-main-cluster-users-integration-k3d
+      annotations:
+        pipeline.clusterprovisioning: "k3d"
+        pipeline.installer: "kyma install"
+        pipeline.platform: "k3d"
+        pipeline.test: "bash resources/cluster-users/files/sar-test.sh"
+        pipeline.trigger: "pr-submit"
+        pipeline.type: "integration"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "pre-main-cluster-users-integration-k3d"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-build-pr: "true"
+        preset-gc-project-env: "true"
+        preset-kyma-guard-bot-github-token: "true"
+        preset-sa-vm-kyma-integration: "true"
+      run_if_changed: '^((resources/cluster-users\S+)(\.[^.][^.][^.]+$|\.[^.][^dD]$|\.[^mM][^.]$|\.[^.]$|/[^.]+$))'
```

Comment:
Can we add here also `/tests/integration/cluster-users` directory? In this way, this job will run if chart or tests are changed.